components/arm.py

```diff
@@ -49,25 +64,31 @@ def __init__(self) -> None:
         self._rotation_motor_right.follow(self.rotation_motor, invert=True)
         self._rotation_motor_right.setIdleMode(CANSparkMax.IdleMode.kCoast)
         self._rotation_motor_right.setInverted(False)
-
-        self.rotation_encoder = self.rotation_motor.getAbsoluteEncoder(
-            SparkMaxAbsoluteEncoder.Type.kDutyCycle
+        self.relative_encoder = SparkMaxEncoderWrapper(self.rotation_motor)
+        # ouput position is conversion factor * motor rotations
+        # should be < 1
+        self.relative_encoder.real_encoder.setPositionConversionFactor(
```

Comment:
A facade shouldn't require breaking the abstraction. It should expose its own method.

Reply:
Or perhaps indeed the (singular) conversion factor should be passed to the constructor, to guarantee the conversion factor is always set. IIRC we had that same design with the Talon encoder wrapper in the past.